contracts/src/types/type-aliases.ts

```diff
@@ -0,0 +1,74 @@
+import { MichelsonMap } from '@taquito/taquito';
+import { BigNumber } from 'bignumber.js';
+
+export type unit = (true | undefined) & { __type: 'unit' };
+
+export type address = string & { __type: 'address' };
```

Comment:
This feels useful across any project that pulls in the [`@taquito/contract-type-generator`](https://github.com/ecadlabs/taquito/pull/721). What are your thoughts on exporting these type-aliases as part of that package?

Reply:
Yes, I actually have a PR to do that. I'll update this.